app/controllers/application_controller.rb

```diff
@@ -4,14 +4,6 @@ class ApplicationController < ActionController::Base
   layout :set_layout
 
   before_action :set_variant
-  before_action :configure_permitted_parameters,
-                if: :devise_controller?
-
-  def configure_permitted_parameters
-    devise_parameter_sanitizer.permit(:account_update) do |u|
-      u.permit(:avatar, :email, :password, :password_confirmation, :current_password)
-    end
-  end
```

Comment:
This was added in the [PR](https://github.com/hpi-swt2/connections-portal/pull/156/files) that introduced profile pictures. I think that everything but the avatar is permitted by default because devise needs it. And because we do not need the avatar anymore I completely removed it to fall back to the default behaviour again.